CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ option (BUILDFTI "Build Funtion Table Interface (FTI) in C sensor libraries" OFF
 option (BUILDSWIGPYTHON "Build swig python modules" ON)
 option (BUILDSWIGNODE "Build swig node modules" ON)
 option (BUILDSWIGJAVA "Build swig java modules" OFF)
+option (BUILDCORDOVABINDING "Build Cordova binding" OFF)
```

Comment:
Maybe "Build Cordova bindings" here will be more accurate

Reply:
Is that okay to still use `BUILDCORDOVABINDING` here? or `BUILDCORDOVABINDINGS"

Reply:
this was just a minor comment, we can disregard it if @Propanu  and @pylbert think it is not relevant